tools/cli/workflow_commands_test.go

```diff
@@ -2088,6 +2088,126 @@ func Test_NewTest(t *testing.T) {
 
 }
 
+func Test_ReplaceRetryPropertiesInErrorMessageWithRetryArguments(t *testing.T) {
+	tests := []struct {
+		originalErrorMessage string
+		expectedErrorMessage string
+	}{
+		{
+			"InitialIntervalInSeconds must be greater than 0 on retry policy.",
+			"retry_interval must be greater than 0 on retry policy.",
+		},
+		{
+			"BackoffCoefficient cannot be less than 1 on retry policy.",
+			"retry_backoff cannot be less than 1 on retry policy.",
+		},
+		{
+			"MaximumIntervalInSeconds cannot be less than 0 on retry policy.",
+			"retry_max_interval cannot be less than 0 on retry policy.",
+		},
+		{
+			"MaximumIntervalInSeconds cannot be less than InitialIntervalInSeconds on retry policy.",
+			"retry_max_interval cannot be less than retry_interval on retry policy.",
+		},
+		{
+			"MaximumAttempts cannot be less than 0 on retry policy.",
+			"retry_attempts cannot be less than 0 on retry policy.",
+		},
+		{
+			"ExpirationIntervalInSeconds cannot be less than 0 on retry policy.",
+			"retry_expiration cannot be less than 0 on retry policy.",
+		},
+		{
+			"MaximumAttempts and ExpirationIntervalInSeconds are both 0. At least one of them must be specified.",
+			"retry_attempts and retry_expiration are both 0. At least one of them must be specified.",
+		},
+	}
+
+	for _, tc := range tests {
+		t.Run(tc.expectedErrorMessage, func(t *testing.T) {
+			err := replaceRetryPropertiesInErrorMessageWithRetryArguments(errors.New(tc.originalErrorMessage))
+			assert.Equal(t, tc.expectedErrorMessage, err.Error())
+		})
+	}
+}
+
+func Test_StartWorkflowHelper_WithRetryArguments(t *testing.T) {
```

Comment:
```suggestion
func Test_StartWorkflowHelper_RetryErrorMapping(t *testing.T) {
```
We are only testing the errors, so lets reflect that in the name

Reply:
@jakobht Renamed the test function.